com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -12,6 +12,7 @@ namespace Unity.Netcode
     /// delegate type <see cref="NetworkSceneManager.SceneEventDelegate"/> uses this class to provide
     /// scene event status.<br/>
     /// <em>Note: This is only when <see cref="NetworkConfig.EnableSceneManagement"/> is enabled.</em><br/>
+    /// <em>*** Do not start new scene events within scene event notification callbacks.</em><br/>
```

Comment:
Should we have (or is there) any mechanism to detect and prevent such scene events ?

Reply:
Why yes there is!
Both `NetworkSceneManager.LoadScene` and `NetworkSceneManager.UnloadScene` return a `SceneEventProgressStatus`.
If you invoke that in the middle of a scene event `NetworkSceneManager` will return `SceneEventProgressStatus.SceneEventInProgress` which tells/reminds the user "You are in the middle of a scene event, let it finish completely before trying to start a new one".
The same thing would happen if you tried to load several scenes all at the same time and called `NetworkSceneManager.LoadScene` within some method.  The first time (iteration 0) it would return `SceneEventProgressStatus.Started` and then for iterations 1-5 it would return `SceneEventProgressStatus.SceneEventInProgress`.

So, this is PR is just to remind users when they are looking through the API that you can't start a new scene event until any current scene events are still in progress.